index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
+<h2 hidden>Game over</h2>
```

Comment:
¿Qué te parece dejar el mensaje de actualización en un tooltip o al hacer hover sobre el texto?”
```suggestion
<h2 hidden>Game over</h2>
```